syntax = "proto3";

/*option java_multiple_files = true;*/
/*option java_package = "io.grpc.examples.helloworld";*/
/*option java_outer_classname = "HelloWorldProto";*/
/*option objc_class_prefix = "HLW";*/

package suresecureivs;

message Empty{
}
message GeneralReply{
  int32 error_code = 1;
  string message = 2;
}

// The device management service definition.
service DeviceMgt {
  rpc GetHealthyStatus (Empty) returns (GeneralReply){}
}

